### DISCUSSION TITLE
Use Frontier

### DISCUSSION THREAD

**Discussion by LilFatFrank** [wevm/wagmi#2171] [General]
How do I use Frontier wallet in injected module along with metamask wallet?
I tried using two separate connectors (injected and metamask) but it doesn't work

**Comment by tmm** [comment]
Unfortunately injected wallets tend to clash over `window.ethereum` so likely what is happening here is MetaMask is overriding Frontier. There's some discussion about this in [EIP-5749](https://eips.ethereum.org/EIPS/eip-5749).
